config/eslint-config/README.md

```diff
@@ -10,42 +10,55 @@ npm install -D @ilo-org/eslint-config
 
 ## Usage
 
-You can use `@ilo-org/eslint-config` in your project by extending it in your
-`eslint` configuration. For example, if we had an `.eslintrc` file:
+`@ilo-org/eslint-config` provides a list of ESLint flat configurations:
 
-```json
-{
-  "extends": ["@ilo-org/eslint-config"]
-}
+| Config Name | Description                           | Internal Extends |
+| ----------- | ------------------------------------- | ---------------- |
+| js          | Base JavaScript config                | ⛔               |
+| recommended | Recommended configuration `js` + `ts` | `js`             |
```

Comment:
I still think that ts or typescript is a better name for this. recommended can mean anything and other opinions that are not related to typescript could also get added. By naming it typescript, we open the options of having more configs that build on top of typescript instead of the recommended getting adapted for every single use case.

From my understanding, you are proposing that project specific opinions should be placed as project level rules right?

Reply:
1. Yes, project-specific rules, should be placed inside the local config, like the PR does for the `twig` package.
2. Since the mentioned config has general rules about `ts` and `js` files, and can be used for any project, I think the name `recommended` is better suited. IMHO by naming it `recommended` we encourage the users to extend it.